artemis-server/src/main/java/org/apache/activemq/artemis/core/config/impl/ConfigurationImpl.java

```diff
@@ -550,6 +550,17 @@ public <T> T convert(Class<T> type, Object value) {
          }
       }, TransformerConfiguration.class);
 
+      beanUtils.getConvertUtils().register(new Converter() {
+         @Override
+         public <T> T convert(Class<T> type, Object value) {
+            List convertedValue = new ArrayList<String>();
+            for (String entry : value.toString().split(",")) {
```

Comment:
I have been following some error prone messages, and this particular line will introduce an error prone warning.

WARNING] /Users/clebertsuconic/work/apache/activemq-artemis/artemis-commons/src/main/java/org/apache/activemq/artemis/utils/uri/URIFactory.java:[112,86] [StringSplitter] String.split(String) has surprising behavior
    (see https://errorprone.info/bugpattern/StringSplitter)
  Did you mean 'List<String> connectorURIS = Splitter.on(',').splitToList(split[0].substring(split[0].indexOf('(') + 1));'?
[

Reply:
thanks!

Reply:
that warn rule must be quite specific, I am not seeing it with -Derrorprone or -Pjdk11to15-errorprone with openjdk 11